tlv/go.sum

```diff
@@ -1,105 +1,30 @@
-github.com/aead/siphash v1.0.1/go.mod h1:Nywa3cDsYNNK3gaciGTWPwHt0wlpNV15vwmswBAUSII=
```

Comment:
Interesting that there's such a big diff in the sum files... Did we just never run `go mod tidy` or did the more recent Go versions change what needs to be validated?